lib/scraped_page_archive/capybara.rb

```diff
@@ -0,0 +1,59 @@
+# Monkey patch capybara poltergiest driver to record http requests automatically.
+require 'capybara/poltergeist'
+require 'scraped_page_archive'
+
+module Capybara::Poltergeist
+  class Driver
+    alias __visit visit
+
+    def sha_url(url)
+      Digest::SHA1.hexdigest url.gsub(/_piref[\d_]+\./, '')
```

Comment:
I'm not quite following why this `gsub` is needed. Perhaps it would be clearer if the Regex was in a named constant somewhere?